database/database.js

```diff
@@ -0,0 +1,33 @@
+import { MongoClient, ServerApiVersion } from "mongodb";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const uri = process.env.URI;
+const dbName = "devdb";
+
+if (!uri) {
+  console.error("env values are not present.");
+  process.exit(1);
+}
+
+const client = new MongoClient(uri, {
+  serverApi: {
+    version: ServerApiVersion.v1,
+    strict: true,
+    deprecationErrors: true,
+  }});
+
+  async function connectToDatabase() {
```

Comment:
Returning client.db this way is a bit of an unusual practice.
But in theory this shouldn't cause a spike in the number of connections (I say this under correction).

Reply:
No further action required now on this point.
You may resolve this conversation after you've made a mental note of my comment.